bin/usage.txt

```diff
@@ -92,6 +92,11 @@ Options:
                               captured by default and sent to the
                               coveralls.io service.
 
+  --typescript --ts           Run .ts files with `node -r ts-node/register`
+                              
+                              In order for this to work,
+                              you must intsall `ts-node` locally.                              
```

Comment:
Typo: `intsall` -> `install`

Also, is a local install necessary given that `ts-node` was added to `package.json`?

Reply:
Ah, missed that it was a dev-dep.

Reply:
there's a good thought to move ts-node to peerDeps, will do today